<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Live">
       <attribute name="title">
        <string>Live</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>Pilot Remaining Time</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLCDNumber" name="lcdPilot">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>177</height>
                </size>
               </property>
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="inputMethodHints">
                <set>Qt::ImhTime</set>
               </property>
               <property name="digitCount">
                <number>8</number>
               </property>
               <property name="value" stdset="0">
                <double>0.000000000000000</double>
               </property>
               <property name="intValue" stdset="0">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QProgressBar" name="progressBarPilot">
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>Fuel Remaining Time</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLCDNumber" name="lcdFuel">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>177</height>
                </size>
               </property>
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="inputMethodHints">
                <set>Qt::ImhTime</set>
               </property>
               <property name="digitCount">
                <number>8</number>
               </property>
               <property name="value" stdset="0">
                <double>0.000000000000000</double>
               </property>
               <property name="intValue" stdset="0">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QProgressBar" name="progressBarFuel">
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="minimumSize">
             <size>
              <width>759</width>
              <height>396</height>
             </size>
            </property>
            <property name="title">
             <string>URL Live Timing</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLineEdit" name="LiveEdit">
               <property name="text">
                <string>http://www.apex-timing.com/live-timing/lemans-karting/index.html</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QScrollArea" name="WebFrameLive">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>1113</width>
                  <height>392</height>
                 </rect>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
       <zorder></zorder>
      </widget>
      <widget class="QWidget" name="Chronos">
       <attribute name="title">
        <string>Chronos</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>Average Lap Time</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QPushButton" name="pushLap">
               <property name="text">
                <string>Set Average Lap Time</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTimeEdit" name="timeLapEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="displayFormat">
                <string>HH:mm:ss</string>
               </property>
               <property name="time">
                <time>
                 <hour>0</hour>
                 <minute>1</minute>
                 <second>45</second>
                </time>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdLap_2">
               <property name="minimumSize">
                <size>
                 <width>351</width>
                 <height>177</height>
                </size>
               </property>
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="inputMethodHints">
                <set>Qt::ImhTime</set>
               </property>
               <property name="frameShape">
                <enum>QFrame::Box</enum>
               </property>
               <property name="lineWidth">
                <number>1</number>
               </property>
               <property name="digitCount">
                <number>8</number>
               </property>
               <property name="value" stdset="0">
                <double>0.000000000000000</double>
               </property>
               <property name="intValue" stdset="0">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>Pilot Remaining Time</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QPushButton" name="pushPilot">
               <property name="text">
                <string>Set Pilot Time</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTimeEdit" name="timePilotEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="displayFormat">
                <string>HH:mm:ss</string>
               </property>
               <property name="time">
                <time>
                 <hour>0</hour>
                 <minute>45</minute>
                 <second>0</second>
                </time>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdPilot_2">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>177</height>
                </size>
               </property>
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="inputMethodHints">
                <set>Qt::ImhTime</set>
               </property>
               <property name="digitCount">
                <number>8</number>
               </property>
               <property name="value" stdset="0">
                <double>0.000000000000000</double>
               </property>
               <property name="intValue" stdset="0">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_8">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>Fuel Remaining Time</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QPushButton" name="pushFuel">
               <property name="text">
                <string>Set Fuel Time</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTimeEdit" name="timeFuelEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="displayFormat">
                <string>HH:mm:ss</string>
               </property>
               <property name="time">
                <time>
                 <hour>2</hour>
                 <minute>20</minute>
                 <second>0</second>
                </time>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdFuel_2">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>177</height>
                </size>
               </property>
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="inputMethodHints">
                <set>Qt::ImhTime</set>
               </property>
               <property name="digitCount">
                <number>8</number>
               </property>
               <property name="value" stdset="0">
                <double>0.000000000000000</double>
               </property>
               <property name="intValue" stdset="0">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Strategy">
       <attribute name="title">
        <string>Strategy</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="minimumSize">
           <size>
            <width>759</width>
            <height>396</height>
           </size>
          </property>
          <property name="title">
           <string>URL Strategy</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLineEdit" name="StrategyEdit">
             <property name="text">
              <string>http://www.apex-timing.com/live-timing/lemans-karting/index.html</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QScrollArea" name="WebFrameStrategy">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>731</width>
                <height>325</height>
               </rect>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1183</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKART">
    <property name="title">
     <string>KART</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionStart_Race"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menuKART"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStart_Race">
   <property name="text">
    <string>Start Race</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
